### ISSUE TITLE
Add Currency library to formatters.ts

### ISSUE THREAD

**Issue by cherry-john** [bstudios/adam-rms#274]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

v2

**Comment by Jbithell** [comment]
I'd be very wary of having two different currency libs running - I think we should get the api to return pretty currencies? @Robert-Watts not sure your views on this

**Comment by cherry-john** [comment]
That would definitely be preferred, although would involve changing v1 api in the short term?

**Comment by Robert-Watts** [comment]
Yeah as a rule of thumb it would be best to get the server to do them. That
being said if this is a short term change with the view it would be looked
at properly in a V2 API then I don't see a problem with this as a temporary
fix as long as it's listed as an issue to fix in the future when we get to
it.

**Comment by Jbithell** [comment]
Yes but that's fine by me, it's a small change

**Comment by Jbithell** [comment]
> Yeah as a rule of thumb it would be best to get the server to do them. That being said if this is a short term change with the view it would be looked at properly in a V2 API then I don't see a problem with this as a temporary fix as long as it's listed as an issue to fix in the future when we get to it.

Cool let's get the api to do it
